presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/TarController.java

```diff
@@ -180,6 +181,11 @@ public void upgradeAsync(URL packageUrl, boolean checkDependencies, boolean pres
                     tarInstall(tempPackage);
                     postInstall();
                     configDeployer.restoreDirectory(tempConfigDir, configDir);
+                    updateProperty(configDir.toAbsolutePath().resolve("node.properties"), "plugin.dir",
```

Comment:
I don't follow.  If configs are preserved why was this wrong in the first place?